src/components/molecules/NavBar/NavBar.tsx

```diff
@@ -293,15 +293,12 @@ const NavBar: React.FC<NavBarPropsType> = ({
                   overlay={ProfilePopover}
                   rootClose={true}
                 >
-                  <div className="navbar-link-profile">
-                    <img
-                      src={profileImage}
-                      className="profile-icon"
```

Comment:
Have you removed `profile-icon` styles?

Reply:
used in `Participant.tsx` and `JazzBar.tsx`